### ISSUE TITLE
Spurious server close error

### ISSUE THREAD

**Issue by rragan** [krakenjs/kraken-js#388]
Having deployed an app and then a bit later deployed it again, I spotted this in the error log (something I’ve seen in other error logs and wondered about).
 
Since nothing else was going on with this app and no traffic, this seems to arise from the shutdown process in the deploy scripts. The error comes from kraken’s shutdown code trying to close the server but the server is already closed. Maybe this error should be handled and silently ignored as the intent of server.close() has already been met – less spurious noise in the logs.
 
```
Error: Not running
  at Server.close (net.js:1237:11)
  at EventEmitter.onshutdown (/.....node_modules/kraken-js/lib/events.js:38:16)
  at EventEmitter.emit (events.js:98:17)
  at process.close (..../node_modules/kraken-js/middleware/shutdown.js:31:13)
  at process.g (events.js:180:16)
  at process.emit (events.js:92:17)
  at Signal.wrap.onsignal (node.js:790:46)
```

**Comment by aredridel** [comment]
I've seen this two in my sample app

**Comment by jasisk** [comment]
Related to #402. tl;dr: `server.close` is called twice for N reason (N = 1 of many).

**Comment by jasisk** [comment]
Closed by #417 #419.
